dex/ws/wslink_test.go

```diff
@@ -115,7 +115,8 @@ func TestWSLink_handleMessageRecover(t *testing.T) {
 		inMsg: make(chan []byte, 1),
 		inErr: make(chan error, 1),
 	}
-	wsLink := NewWSLink(dex.IPKey{}, "addr", conn, time.Second, inMsgHandler, tLogger)
+	ipk := dex.IPKey{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 255, 255 /* ipv4 */, 127, 0, 0, 1}
+	wsLink := NewWSLink(ipk.String(), conn, time.Second, inMsgHandler, tLogger)
```

Comment:
As I mentioned, dex.IPKey is only needed in server/comms, but this is here for an example in the test.